server/templates/details.html.tmpl

```diff
@@ -65,14 +65,19 @@
             {{template "result-predicates-details" .}}
           </div>
         {{end}}
-        {{if (hasRequires .Requires)}}
+        {{if (hasActors .Requires)}}
           <div class="pt-2">
-          {{template "result-requires-details" .}}
+          {{template "result-actors-details" .}}
           </div>
         {{end}}
-        {{if (hasRequiresPermissions .Requires)}}
+        {{if (hasPatterns .)}}
           <div class="pt-2">
-          {{template "result-requires-permissions-details" .}}
+          {{template "result-patterns-details" .}}
```

Comment:
I think it would make sense to reorder these templates so it reads something like:

> This rule needs N approvals from people in these groups or people with these permissions. Approvals can use these patterns.

As currently structured, it kind of implies that the sections are independent so that the pattern section doesn't also require the approvers have certain team membership/permissions.